configuration/contrib/collectd/collectd.conf

```diff
@@ -512,7 +512,7 @@ LoadPlugin mqtt
 </Plugin>
 
 #<Plugin disk>
-#	Disk "hda"
+#	Disk "had"
```

Comment:
This needs to be reverted.

How can we avoid this kind of false negative? `default.extend-words` ? `extend-exclude` ? In a configuration file, as here, I prefer the later.

Reply:
Good catch!

I will add an exception for "hda".